win32/GNUmakefile

```diff
@@ -583,15 +583,14 @@ MINIBUILDOPT    += -D__USE_MINGW_ANSI_STDIO
 endif
 
 
-# If you are using GCC, 4.3 or later by default we add the -fwrapv option.
-# See https://github.com/Perl/perl5/issues/13690
-#
-GCCWRAPV := $(shell if "$(GCCVER1)"=="4" (if "$(GCCVER2)" geq "3" echo define) else if "$(GCCVER1)" geq "5" (echo define))
+# Previously, if you were using GCC, 4.3 or later we addeed the -fwrapv option.
+# See https://github.com/Perl/perl5/issues/13690 and 18739
+# However, as documented in README.win32 our minimum Win32 gcc version is 3.4.5.
+# As that supports -fwrapv it's both simpler and more correct to unconditionally
+# add it to the flags:
 
-ifeq ($(GCCWRAPV),define)
 BUILDOPT        += -fwrapv
 MINIBUILDOPT    += -fwrapv
-endif
```

Comment:
Good points, and I'd not thought of this. Agree, this is better than what I did.

Given that I don't have access to a Win32 system to test this, and I think that you do, and I think that testing the non-default case requires doing something interactively

1. Could you implement this and test that it works
2. Push it to a branch to replace mine, so that (at least one of) @steve-m-hay @xenu or @sisyphus can check it (better than me)
3. If someone else says yes and no-one says no, it goes to blead.

I feel like I'm turning into a project manager and it's not an enjoyable thought. :-/